pkg/config/user_config.go

```diff
@@ -305,7 +305,7 @@ type KeybindingUniversalConfig struct {
 	ConfirmInEditor              string   `yaml:"confirmInEditor"`
 	Remove                       string   `yaml:"remove"`
 	New                          string   `yaml:"new"`
-	Edit                         string   `yaml:"edit"`
+	Edit                         string   `yaml:"edit" jsonschema:"example=vim,example=nvim,example=emacs,example=nano,example=vscode,example=sublime,example=bbedit,example=kakoune,example=helix,example=xcode"`
```

Comment:
This is wrong here; this is the keybinding for the "edit" command. I suppose you wanted to attach these to `os.editPrefix`?

Reply:
fixed in 34b0f495e4f784b5918dc0fd2170dcd1c299fec1